line-bot-servlet/README.md

```diff
@@ -0,0 +1,4 @@
+line-bot-servlet
+====
+
+This module is deprecated and will be removed next release.
```

Comment:
How about annotating with `@Deprecated` and adding comments on the related classes?

Reply:
Good idea. Done.